Testing/UnitTests/xASL_ut_function_xASL_adm_CheckFileCount.m

```diff
@@ -26,12 +26,9 @@
 testTime = tic;
 
 % Create three test files in test directory
-fid = fopen(fullfile(TestRepository,'UnitTesting','working_directory','test_1.txt'), 'wt');
-fclose(fid);
-fid = fopen(fullfile(TestRepository,'UnitTesting','working_directory','test_2.txt'), 'wt');
-fclose(fid);
-fid = fopen(fullfile(TestRepository,'UnitTesting','working_directory','test_3.txt'), 'wt');
-fclose(fid);
+fid1 = fopen(fullfile(TestRepository,'UnitTesting','working_directory','test_1.txt'), 'wt');
```

Comment:
Please add the function name in the commit title ;)
Weird why this doesn't work. In Linux there is a much easier way to create an empty file, which is called "touching" a file": `system('touch test_1.txt');`

Reply:
Yeah, I always try to avoid system statements wherever possible, because then we have to take these Windows/Unix differences into account.